### PR TITLE
Feature/inline css

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- fixing phpstan
- fixing phpcs
- merging @mbmjertan PR
- updating libs and flibs to new release
- fixing issue with new release

### CODE REVIEW

**Comment by iruzevic** [comment]
@mbmjertan is there a chance you can fix 2 test issues before we release

tnx

File: tests/Validation/ValidatorTest.php [hunk 1]
```diff
@@ -0,0 +1,856 @@
...
+				'fileIsRequired' => true,
+			],
+			[
+				'component' => 'file',
+				'fileName' => 'file-no-validation',
+				'fileId' => 'file-no-validation',
+			],
+		],  
+	],
+]);
```
Comment (volfkarlo) [inline tests/Validation/ValidatorTest.php:856]: Missing empty line

File: tests/Validation/ValidationSettingsTest.php [hunk 2]
```diff
@@ -35,7 +36,210 @@ public function __construct(LabelsInterface $labels) {
...
+			'component' => 'input',
+			'inputName' => 'es-forms-mailerErrorEmailConfirmationSend-HR',
+			'inputId' => 'es-forms-mailerErrorEmailConfirmationSend-HR',
+			'inputFieldLabel' => 'MailerErrorEmailConfirmationSend',
+			'inputPlaceholder' => 'Confirmation e-mail was not sent due to unknown issue. Please try again.',
+			'inputValue' => 'es-forms-mailerErrorEmailConfirmationSend-HR',
+		]
+	];
+	expect($this->validationSettings->getSettingsGlobalData())->toBe($expected);
+});
```
Comment (volfkarlo) [inline tests/Validation/ValidationSettingsTest.php:245]: Missing empty line

File: tests/Helpers.php [hunk 3]
```diff
@@ -34,6 +47,73 @@ function setupMocks() {
...
+			return $key;
+		}
+
+		if ($envValue === 'unset') {
+			return '';
+		}
+
+		return $envValue;
+	});
 }
```
Comment (volfkarlo) [inline tests/Helpers.php:119]: two lines at the end

File: tests/Hooks/FiltersTest.php [hunk 4]
```diff
@@ -67,3 +67,16 @@
...
+
+test('getValidationSettingsFilterName will return correct filter name', function () {
+	$filter = $this->filters::getValidationSettingsFilterName('failMimetypeValidationWhenFileNotOnFS');
+
+	$this->assertSame('es_forms_validation_force_mimetype_from_fs', $filter);
+});
+
+test('getValidationSettingsFilterName will throw error if wrong filter name or name is provided.', function () {
+	$this->filters::getGeolocationFilterName('wrong');
+})->throws(MissingFilterInfoException::class);
```
Comment (volfkarlo) [inline tests/Hooks/FiltersTest.php:82]: no empty line

File: tests/Hooks/FiltersTest.php [hunk 5]
```diff
@@ -67,3 +67,16 @@
 test('getGeolocationFilterName will throw error if wrong filter name or name is provided.', function () {
 	$this->filters::getGeolocationFilterName('wrong');
 })->throws(MissingFilterInfoException::class);
+
+
```
Comment (volfkarlo) [inline tests/Hooks/FiltersTest.php:71]: double new line

File: tests/Mailer/MailerTest.php [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+	expect(getenv('test_wp_mail_last_call'))->toBe($expected['wp_mail_call']);
+})->with([
+	[
+		[
+			'return' => false,
+			'wp_mail_call' => '{"to":"","subject":"","message":"","headers":["Content-Type: text\/html; charset=UTF-8","From: sender@example.com"],"attachments":[]}'
+		],
+		'', '', '',
+	],
+]);
```
Comment (volfkarlo) [inline tests/Mailer/MailerTest.php:134]: no empty line

File: tests/Permissions/PermissionsTest.php [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        'edit_eightshift_forms',
+        'read_eightshift_forms',
+        'delete_eightshift_forms',
+        'edit_eightshift_formss',
+        'edit_others_eightshift_formss',
+        'delete_eightshift_formss',
+        'publish_eightshift_formss',
+        'read_private_eightshift_formss',
+    ]);
+});
```
Comment (volfkarlo) [inline tests/Permissions/PermissionsTest.php:38]: no empty line

**Review by volfkarlo — CHANGES_REQUESTED** [review]
Minor changes/suggestions.
Well done 👏 🎉

**Comment by mbmjertan** [comment]
@iruzevic yes, sure, I'll check what's going on there

File: src/Integrations/Clearbit/ClearbitClient.php [hunk 8]
```diff
@@ -92,23 +92,23 @@ public function getApplication(string $emailKey, array $params, array $mapData):
...
+		if (\is_wp_error($response)) {
 			return [
 				'status' => 'error',
 				'code' => 400,
 				'message' => $this->getErrorMsg('submitWpError'),
 			];
 		}
 
-		$code = $response['response']['code'] ?? 200;
+		$code = $response['response']['code'] ? $response['response']['code'] : 200;
```
[the remark this answers is not in the document]
Comment (mbmjertan) [reply src/Integrations/Clearbit/ClearbitClient.php:103]: This is discouraged by Eightshift Coding Standards, IIRC.

File: src/Integrations/Greenhouse/GreenhouseClient.php [hunk 9]
```diff
@@ -51,7 +51,7 @@ class GreenhouseClient implements ClientInterface
 	 */
 	public function getItems(bool $hideUpdateTime = true): array
 	{
-		$output = get_transient(self::CACHE_GREENHOUSE_ITEMS_TRANSIENT_NAME) ?: []; // phpcs:ignore WordPress.PHP.DisallowShortTernary.Found
+		$output = \get_transient(self::CACHE_GREENHOUSE_ITEMS_TRANSIENT_NAME) ?: []; // phpcs:ignore WordPress.PHP.DisallowShortTernary.Found
```
[the remark this answers is not in the document]
Comment (mbmjertan) [reply src/Integrations/Greenhouse/GreenhouseClient.php:54]: 😄

File: src/Blocks/assets/styles/application-blocks-frontend.scss [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+// @import './../../../../assets/styles/parts/shared';
+
+// Register Wrapper block styles.
+// @import './../../wrapper/wrapper-frontend.scss';
+
+// Find all components and require component_name-style.scss inside it.
+// @import './../../components/*/*-frontend.scss';
+
+// Find all blocks and require block_name-editor.scss inside it.
+// @import './../../custom/*/*-frontend.scss';
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply src/Blocks/assets/styles/application-blocks-frontend.scss:21]: these are placeholders like in the boilerplate.

**Review by mbmjertan — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Pull request merged**